jedi.inc

```diff
@@ -1214,6 +1217,213 @@
+
+  {$DEFINE BDS8}
+  {$DEFINE COMPILER8}
+  {$DEFINE DELPHI8}
+  {$DEFINE DELPHICOMPILER8}
+
+  {$DEFINE BDS9}
+  {$DEFINE COMPILER9}
+  {$DEFINE DELPHI9}
+  {$DEFINE DELPHICOMPILER9}
+
+  {$DEFINE BDS10}
+  {$DEFINE COMPILER10}
+  {$DEFINE DELPHI10}
+  {$DEFINE DELPHICOMPILER10}
+  {$DEFINE RTL100_UP}
+
+  {$DEFINE BDS11}
+  {$DEFINE COMPILER11}
+  {$DEFINE DELPHI11}
+  {$DEFINE DELPHICOMPILER11}
+  {$DEFINE RTL110_UP}
+
+  {$DEFINE BDS12}
+  {$DEFINE COMPILER12}
+  {$DEFINE DELPHI12}
+  {$DEFINE DELPHICOMPILER12}
+  {$DEFINE RTL120_UP}
+
+  {$DEFINE BDS14}
+  {$DEFINE COMPILER14}
+  {$DEFINE DELPHI14}
+  {$DEFINE DELPHICOMPILER14}
+  {$DEFINE RTL140_UP}
+
+  {$DEFINE BDS15}
+  {$DEFINE COMPILER15}
+  {$DEFINE DELPHI15}
+  {$DEFINE DELPHICOMPILER15}
+  {$DEFINE RTL150_UP}
+
+  {$DEFINE BDS16}
+  {$DEFINE COMPILER16}
+  {$DEFINE DELPHI16}
+  {$DEFINE DELPHICOMPILER16}
+  {$DEFINE RTL160_UP}
+
+  {$DEFINE BDS17}
+  {$DEFINE COMPILER17}
+  {$DEFINE DELPHI17}
+  {$DEFINE DELPHICOMPILER17}
+  {$DEFINE RTL170_UP}
+
+  {$DEFINE BDS18}
+  {$DEFINE COMPILER18}
+  {$DEFINE DELPHI18}
+  {$DEFINE DELPHICOMPILER18}
+  {$DEFINE RTL180_UP}
+
+  {$DEFINE BDS19}
+  {$DEFINE COMPILER19}
+  {$DEFINE DELPHI19}
+  {$DEFINE DELPHICOMPILER19}
+  {$DEFINE RTL190_UP}
+
+  {$DEFINE BDS20}
+  {$DEFINE COMPILER20}
+  {$DEFINE DELPHI20}
+  {$DEFINE DELPHICOMPILER20}
+  {$DEFINE RTL200_UP}
+
+  {$DEFINE BDS21}
+  {$DEFINE COMPILER21}
+  {$DEFINE DELPHI21}
+  {$DEFINE DELPHICOMPILER21}
+  {$DEFINE RTL210_UP}
+
+  {$DEFINE BDS22}
+  {$DEFINE COMPILER22}
+  {$DEFINE DELPHI22}
+  {$DEFINE DELPHICOMPILER22}
+  {$DEFINE RTL220_UP}
+
+  {$DEFINE BDS23}
+  {$DEFINE COMPILER23}
+  {$DEFINE DELPHI23}
+  {$DEFINE DELPHI_SEATTLE} // synonym to DELPHI23
+  {$DEFINE DELPHICOMPILER23}
+  {$DEFINE RTL230_UP}
+
+  {$DEFINE BDS24}
+  {$DEFINE COMPILER24}
+  {$DEFINE DELPHI24}
+  {$DEFINE DELPHI_BERLIN} // synonym to DELPHI24
+  {$DEFINE DELPHICOMPILER24}
+  {$DEFINE RTL240_UP}
+
+  {$DEFINE BDS25}
+  {$DEFINE COMPILER25}
+  {$DEFINE DELPHI25}
+  {$DEFINE DELPHI_TOKYO} // synonym to DELPHI25
+  {$DEFINE DELPHICOMPILER25}
+  {$DEFINE RTL250_UP}
+
+  {$DEFINE BDS26}
+  {$DEFINE COMPILER26}
+  {$DEFINE DELPHI26}
+  {$DEFINE DELPHI_RIO} // synonym to DELPHI26
+  {$DEFINE DELPHICOMPILER26}
+  {$DEFINE RTL260_UP}
+
+  {$DEFINE BDS27}
+  {$DEFINE COMPILER27}
+  {$DEFINE DELPHI27}
+  {$DEFINE DELPHI_SYDNEY} // synonym to DELPHI27
+  {$DEFINE DELPHICOMPILER27}
+  {$DEFINE RTL270_UP}
+
+  {$DEFINE BDS28}
+  {$DEFINE COMPILER28}
+  {$DEFINE DELPHI28}
+  {$DEFINE DELPHI_ALEXANDRIA} // synonym to DELPHI28
+  {$DEFINE DELPHICOMPILER28}
+  {$DEFINE RTL280_UP}
+
+  {$DEFINE BDS29}
+  {$DEFINE COMPILER29}
+  {$DEFINE DELPHI29}
+  {$DEFINE DELPHI_ATHENS} // synonym to DELPHI29
+  {$DEFINE DELPHICOMPILER29}
+  {$DEFINE RTL290_UP}
+
+  {$DEFINE BDS37}
+  {$DEFINE COMPILER37}
+  {$DEFINE DELPHI37}
+  {$DEFINE DELPHI_FLORENCE} // synonym to DELPHI37
+  {$DEFINE DELPHICOMPILER37}
+  {$DEFINE RTL370_UP}
```

Comment:
Please choose only one version of Delphi for Oxygene to be compatible with. As it is now, you claim that Oxgene is all those versions at once, which it cannot be.
From memory, I believe you should stick with BDS22 or 23, later versions have syntaxes that may not be supported.